service/tablet_allocator.cc

```diff
@@ -1228,6 +1228,7 @@ class load_balancer {
 
     tablet_count_and_reason tablet_count_from_min_per_shard_tablet_count(const schema& s,
             const std::unordered_map<sstring, unsigned>& shards_per_dc,
+            const std::unordered_map<endpoint_dc_rack, unsigned>& shards_per_rack,
```

Comment:
nit: typos in git commit messages:
confied -> confined
determing -> determining